app/views/site/faq.scala

```diff
@@ -350,15 +346,14 @@ object faq {
         ),
         question(
           "make-a-bot",
-          "Make a Lichess bot?",
+          howToMakeABot.txt(),
           p(
-            "To learn how to create a ",
-            a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess bot"),
-            ", please read ",
-            a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")(
-              "this blog post"
+            toLearnHowToMakeABot(
+              a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("Lichess bot")
             ),
-            "."
+            pleaseRead(
+              a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")("blog post")
+            )
```

Comment:
this is still translation lego. Anyway I don't know how useful it is to translate a sentence that only links to 2 english blog posts.

Reply:
Do you mean the scala file should in the end look something like this?
```scala
p(
    toLearnHowToMakeABot(
         a(href := "https://lichess.org/blog/WvDNticAAMu_mHKP/welcome-lichess-bots")("bot"),  
         a(href := "https://lichess.org/@/thibault/blog/how-to-create-a-lichess-bot/FuKyvDuB")("blog post")
    )
)
```

and the xml like this 
```xml
  <string name="toLearnHowToMakeABot">To learn how to create a Lichess %s, please read this %s.</string>
```

Reply:
Yeah, something like that. You'd probably also want translations for "blog post" and maybe "Lichess bot" as well. It's also possible to add context like `<string name="..." context="%1$s will be replaced with the translation of 'blog post">`. Also, you need to use `%1$s` and `%2$s` if there are multiple placeholders.

Reply:
This kind of complicated translations with translated placeholders often confuse translators, and we get broken content. I'm not sure it's worth it tbh.

I would just keep that untranslated. All it does is link developer resources that are written in english.

Reply:
@ornicar if you would leave it like this, the pull request can be merged or not?

Reply:
Yes, if you remove this translation